src/read_pool.rs

```diff
@@ -292,14 +313,215 @@ impl From<Vec<FuturePool>> for ReadPool {
     }
 }
 
-pub struct ReadPoolConfigManager(pub ReadPoolHandle, pub SyncSender<usize>);
+struct ReadPoolCpuTimeTracker {
+    pool_name: String,
+    // the total time duration of each thread busy with handling tasks. This time also includes
+    // the time when the threads are off-cpu, so it might be much higher than the actual cpu time.
+    prev_total_task_handling_time_us: u64,
+    prev_check_time: Instant,
+    prev_cpu_per_sec: f64,
+}
+
+impl ReadPoolCpuTimeTracker {
+    fn new(pool_name: String) -> Self {
+        let prev_check_time = Instant::now_coarse();
+        let prev_total_task_handling_time_us = MULTILEVEL_LEVEL_ELAPSED
```

Comment:
can we use static metrics

Reply:
Yes, It's better to use a static metric.  @Connor1996  PTAL again, thank you